src/main/java/uk/gov/companieshouse/company_appointments/interceptor/AuthenticationHelperImpl.java

```diff
@@ -109,14 +106,14 @@ public boolean isRoleAuthorised(HttpServletRequest request, String role) {
         return ArrayUtils.contains(roles, role);
     }
 
-    @Override
-    public String getAuthorisedKeyRoles(HttpServletRequest request) {
-        return getRequestHeader(request, ERIC_AUTHORISED_KEY_ROLES);
+    public String[] getApiKeyPrivileges(HttpServletRequest request) {
+        return request.getHeader("ERIC-Authorised-Key-Privileges")
```

Comment:
ought this string be made a constant, as has been done for other strings in here?

Reply:
done